### ISSUE TITLE
Failed to minify the code from this file:

### ISSUE THREAD

**Issue by actelligent-team** [ealush/butter-toast#104]
Thanks for good toast.
I am getting this error when build app only.
It works well in development.

![screen shot 2019-01-25 at 8 18 09 am](https://user-images.githubusercontent.com/42307840/51714861-f54b5f80-2079-11e9-8143-959242afb9ed.png)

I am using     "butter-toast": "^3.2.1"

What version is stable?

Thanks

**Comment by vikomte** [comment]
UP !

**Comment by vikomte** [comment]
Same problem..

**Comment by actelligent-team** [comment]
Isn't there any stable version?

**Comment by ealush** [comment]
Hey @actelligent-team & @vikomte, sorry that you experienced this issue. Researching this issue a bit, it seems to be related to an outdated version of `react-scripts` in `create-react-apps` projects [see this](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify).

This issue gave me the chance to make some updates to all the dependencies of the project, and also include node_modules in the transpilation process to prevent this issue from happening on other platforms.

I just tested with latest `create-react-app` and `react scripts` and it seems to be working fine. Can you please confirm?

**Comment by actelligent-team** [comment]
Hi @ealush 
Thanks for your help, unfortunately, it's not working :(

These are packages I am using.

``` "dependencies": {
    "@babel/runtime": "7.0.0-beta.55",
    "@material-ui/core": "1.4.3",
    "@material-ui/icons": "2.0.1",
    "@types/googlemaps": "3.30.11",
    "@types/markerclustererplus": "2.1.33",
    "ajv": "6.5.2",
    "axios": "^0.18.0",
    "butter-toast": "^3.2.2",
    "chance": "^1.0.16",
    "chartist": "0.10.1",
    "halogenium": "^2.2.3",
    "history": "4.7.2",
    "match-sorter": "^2.3.0",
    "moment": "2.22.2",
    "node-sass-chokidar": "1.3.3",
    "npm-run-all": "4.1.3",
    "perfect-scrollbar": "1.4.0",
    "react": "16.4.1",
    "react-big-calendar": "0.19.2",
    "react-bootstrap-sweetalert": "4.4.1",
    "react-chartist": "0.13.1",
    "react-datetime": "2.15.0",
    "react-device-detect": "^1.6.1",
    "react-dom": "16.4.1",
    "react-google-maps": "9.4.5",
    "react-jvectormap": "0.0.3",
    "react-nouislider": "2.0.1",
    "react-redux": "5.0.7",
    "react-router-dom": "4.3.1",
    "react-scripts": "1.1.4",
    "react-swipeable-views": "0.12.15",
    "react-table": "^6.8.6",
    "react-tagsinput": "3.19.0",
    "react-transition-group": "^2.5.1",
    "redux": "4.0.0",
    "redux-persist": "5.9.1",
    "redux-thunk": "2.3.0"
  },```

I am getting this error.
`Failed to minify the code from this file:

        ./node_modules/butter-toast/dist/butter-toast.min.js:1:362`

What I am doing wrong?
Best Regards

**Comment by actelligent-team** [comment]
Hi @ealush
I have solved this by upgrading react-scripts to 2.1.3.

Thanks for your support!
